hawkular-alerts-rest-tests/src/test/groovy/org/hawkular/alerts/rest/AbstractITestBase.groovy

```diff
@@ -46,5 +45,11 @@ class AbstractITestBase {
             .encodeToString("$testUser:$testPasword".getBytes("utf-8"))
          */
         client.defaultRequestHeaders.Authorization = "Basic amRvZTpwYXNzd29yZA=="
+
+        /*
+            This is used for non-bus scenarios.
+            In Bus scenarios this property is skipped
```

Comment:
Indeed it's skipped.
Hawkular Accounts injects a Persona object and it uses a Hawkular-Persona header.
What hawkular-alert-rest does is to detect if we have accounts or not.
If we have accounts, then it fills the internal tenantId with Persona.getId().
If not present, then it check the Hawkular-Tenant header and fills the tenantId.
It this case, the -rest.war is not coupled with hawkular-accounts and it can be secured with basic, keycloak or other mechanisms.